<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Metodo:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmb_Metodo"/>
    </item>
    <item>
     <widget class="QWidget" name="renderWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMesh">
    <property name="title">
     <string>Mesh</string>
    </property>
    <addaction name="actionView_Mesh"/>
    <addaction name="actionOpen_Mesh"/>
    <addaction name="actionSave_Mesh"/>
   </widget>
   <widget class="QMenu" name="menuSketch">
    <property name="title">
     <string>Sketch</string>
    </property>
    <addaction name="actionView_Sketch"/>
    <addaction name="actionLoad_Sketch"/>
    <addaction name="actionSave_Sketch"/>
   </widget>
   <widget class="QMenu" name="menuConfigurations">
    <property name="title">
     <string>Configurations</string>
    </property>
    <addaction name="actionSaveImageAs"/>
   </widget>
   <addaction name="menuMesh"/>
   <addaction name="menuSketch"/>
   <addaction name="menuConfigurations"/>
  </widget>
  <action name="actionSaveImageAs">
   <property name="text">
    <string>Save Image as ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionView_Mesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Mesh</string>
   </property>
  </action>
  <action name="actionOpen_Mesh">
   <property name="text">
    <string>Load Mesh ...</string>
   </property>
  </action>
  <action name="actionLoad_Sketch">
   <property name="text">
    <string>Load Sketch ...</string>
   </property>
  </action>
  <action name="actionSave_Mesh">
   <property name="text">
    <string>Save Mesh ...</string>
   </property>
  </action>
  <action name="actionSave_Sketch">
   <property name="text">
    <string>Save Sketch ...</string>
   </property>
  </action>
  <action name="actionView_Sketch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Sketch</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
